src/main/resources/anet-schema.yml

```diff
@@ -524,11 +524,48 @@ properties:
             additionalProperties:
               "$ref": "#/$defs/customField"
 
+      attachment:
+        type: object
+        additionalProperties: false
+        required: [shortLabel, shortName, description, classification]
```

Comment:
`disabled` should also be required, else it's unclear what it means if the field is not in the dictionary (and the code currently failes with a `java.lang.NullPointerException: Cannot invoke "java.lang.Boolean.booleanValue()" because "userPermission" is null` error in such a case).